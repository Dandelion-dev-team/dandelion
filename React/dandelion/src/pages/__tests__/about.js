import React from "react"
import renderer from "react-test-renderer"

import About from "../About"

jest.mock('localStorage');

describe("About", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<About />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})