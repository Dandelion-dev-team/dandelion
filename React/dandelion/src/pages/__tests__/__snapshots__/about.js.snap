// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<div>
  <header>
    <div
      className="container"
    >
      <div
        className="inner-header"
      >
        <div
          className="logo"
        >
          <a
            exact={true}
            href="/"
          >
            <img
              src="test-file-stub"
            />
          </a>
        </div>
        <div
          className="navigation"
        >
          <nav>
            <a
              href="/data"
            >
              Data
            </a>
            <a
              href="/map"
            >
              Map
            </a>
            <a
              href="/about"
            >
              About
            </a>
            <a
              href="/signin"
            >
              Sign In
            </a>
          </nav>
        </div>
      </div>
    </div>
  </header>
  <div
    className="about"
  >
    <div
      className="container"
    >
      <div
        className="hero-section"
      >
        <div
          className="heading"
        >
          <h3>
            The Dandelion School Growing Initiative
          </h3>
          <p>
            500 schools. 200 growing cubes. The largest community-led growing experiment ever undertaken in Scotland.
          </p>
        </div>
        <div
          className="content"
        />
      </div>
    </div>
  </div>
</div>
`;
